docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+version: "3.2"
```

Comment:
if you want use the template structure put `docker-compose` and related stuff in the `/docker` folder
else ignore my comment.